### DISCUSSION TITLE
Error on production Environment "ReferenceError: deleteEntry is not defined"

### DISCUSSION THREAD

**Discussion by infiniteroles** [Laravel-Backpack/community-forum#1073] [Q&A (Help)]
I’m having problems in my production environment with the delete action. This happens in all CRUDs. When I click the “Delete” button, I get the following error in the console: ReferenceError: deleteEntry is not defined. However, in previous environments, everything works fine.

Does this sound familiar to you? Any idea what might be happening?

thank you very much

**Comment by pxpm** [comment]
Hey @infiniteroles 

Are the dev and prod versions the same ? You can check with `php artisan backpack:version`, they should be, but 🤷‍♂️ 

Also check in the developer console network tab if you don't have any file loading issue. 

Cheers

**Reply by infiniteroles** [reply]
Hi, yes, same versions:

```
### LARAVEL VERSION:
10.41.0.0


### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.2.2
backpack/crud: 6.6.1
backpack/generators: v4.0.3
backpack/pro: 2.1.2
backpack/theme-tabler: 1.2.0
```

And this is what i see indevelopers console:
```
Mixed Content: The page at 'https://mydomain.com/admin/user' was loaded over HTTPS, but requested an insecure script 'http://mydomain.com/storage/basset/backpack/crud/buttons/delete-button-es-7143bdd8.js?ebe4b0ad4676'. This request has been blocked; the content must be served over HTTPS.`
```

I added this to my AppServiceProvider file:
```
 public function register()
    {
        if ($this->app->environment(['production'])) {
            URL::forceScheme('https');
        }
    }
```


Thak you very much,

**Reply by karandatwani92** [reply]
What is set in the `.env`. It should be:
```
APP_URL=https://mydomain.com
ASSET_URL=https://mydomain.com
```

**Reply by infiniteroles** [reply]
oohhm I had http url in mi env file :-(

Thank you very much for your help!!!!
